app/messages/service_credential_binding_create_message.rb

```diff
@@ -0,0 +1,33 @@
+module VCAP::CloudController
+  class ServiceCredentialBindingCreateMessage < MetadataBaseMessage
+    register_allowed_keys [:type, :name, :relationships]
+    validates_with NoAdditionalKeysValidator, RelationshipValidator
+    validates :type, allow_blank: false, inclusion: {
+      in: %w(key app),
```

Comment:
If we don't include this here at all we won't need to think about it until key creation is implemented (after app bindings for MSI). Might make the logic on the controller easier? (not a big deal anyway)

Reply:
The story asks for rejection of `type=key` when creating a binding from a UPSI, so I guess we need this? It would make the code simpler if `type` wasn't required at this point at all 🤔